dashboard/dashboard-slow-query.md

```diff
@@ -60,6 +60,10 @@ Click any item in the list to display detailed execution information of the slow
 
 ### SQL
 
+> **Note:**
+>
+> The maximum length of the query is limited by the [`tidb_stmt_summary_max_sql_length`](/system-variables.md#tidb_stmt_summary_max_sql_length-new-in-v40) system variable.
```

Comment:
```suggestion
> The maximum length of the `Query` column is limited by the [`tidb_stmt_summary_max_sql_length`](/system-variables.md#tidb_stmt_summary_max_sql_length-new-in-v40) system variable.
```

Reply:
```suggestion
> The maximum length of the query recorded in the `Query` column is limited by the [`tidb_stmt_summary_max_sql_length`](/system-variables.md#tidb_stmt_summary_max_sql_length-new-in-v40) system variable.
```

This is limiting what we put into this column. It is not limiting the column itself. Not an important difference...